packages/xrpl/test/models/escrowFinish.test.ts

```diff
@@ -20,6 +20,9 @@ describe('EscrowFinish', function () {
       Condition:
         'A0258020E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855810100',
       Fulfillment: 'A0028000',
+      CredentialIDs: [
```

Comment:
The `w/o optional` test should remove this field

Reply:
There should also be more tests here for codecov for all the `CredentialIDs` validations

Reply:
added these changes